manager/controllers/baselines.go

```diff
@@ -26,7 +26,7 @@ type BaselinesDBLookup struct {
 
 type BaselineItemAttributes struct {
 	Name    string `json:"name" csv:"name" query:"bl.name" gorm:"column:name"`
-	Systems int    `json:"systems" csv:"systems" query:"sp.baseline_id" gorm:"column:systems"`
+	Systems int    `json:"systems" csv:"systems" query:"systems" gorm:"column:systems"`
```

Comment:
I am thinking should we return description, toDate info (all updatable columns) from 'get' request in /baselines. Because if we want to update a baseline, we should show all the data that is updatable.

Reply:
Yes, I would like to extend api description. I will do that in a separate pr.